js/src/forum/getTheme.js

```diff
@@ -0,0 +1,45 @@
+import { Themes, Constants } from '../common/config';
+
+export default function GetTheme(user) {
+    const PerDevice = user.preferences().fofNightMode_perDevice ? user.preferences().fofNightMode_perDevice : false;
+
```

Comment:
This file has many preferences that are accessed multiple times - `fofNightMode_themeType ` and `fofNightMode`. These should be put into variables to clean up the code.